cilium-health/cmd/root.go

```diff
@@ -82,6 +82,7 @@ func init() {
 	flags.BoolP("json", "j", false, "Format as JSON")
 	// TODO GH #2083 Hide until all commands support JSON output
 	flags.MarkHidden("json")
+	// TODO: Support simplified formatting (Strip RFC3339 in nested run)
```

Comment:
the logrus formatter should be configurable per package. Can you create a GH issue for this and reference it here? I prefer having all TODO's ref. GH issues.

Reply:
Done: #2567, will link